platform/x11/os_x11.cpp

```diff
@@ -369,7 +369,7 @@ Error OS_X11::initialize(const VideoMode &p_desired, int p_video_driver, int p_a
 	}
 
 	// disable resizable window
-	if (!current_videomode.resizable) {
+	if (!current_videomode.resizable && !current_videomode.fullscreen) {
```

Comment:
What if you init with `!resizable && !fullscreen`, and later on toggle windowed mode?

Reply:
Just tried it and setting windowed mode later works fine.